### PR TITLE
Display message to users affected by change password bug #1113

### PR DESCRIPTION
**Pull request by bartekn**
Don't merge until:
* [x] stellar/stellar-wallet-js-sdk#16 is merged
* [x] stellar/stellar-wallet#66 is merged
* [x] stellar/stellar-wallet#66 is deployed

### CODE REVIEW

File: app/templates/flash-message-change-password-bug.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<div class="pull-right">
+    <button class="btn btn-default btn-primary submit" ng-click="$parent.startRecoveryProcess()">Go to Settings</button>
+</div>
+<span class="flash-info">Your recovery code has an error.</span>
```
Comment (nullstyle) [inline app/templates/flash-message-change-password-bug.html]: We should probably get a link to a help desk faq entry, or some more explanatory info here.

File: app/scripts/controllers/login-v2-controller.js [hunk 2]
```diff
@@ -58,34 +58,50 @@ angular.module('stellarClient').controller('LoginV2Ctrl', function($scope, $stat
...
+          version: 2,
+          id: wallet.getWalletId(),
+          key: wallet.getWalletKey(),
+          keychainData: wallet.getKeychainData(),
+          mainData: wallet.getMainData(),
+          walletV2: wallet
+        }));
+      })
+      .then(function(wallet) {
+        var bugDeploy   = new Date('2014-11-17');
```
Comment (nullstyle) [inline app/scripts/controllers/login-v2-controller.js]: I would prefer that we extract this into it's own method, with a jsdoc block that further explains the issue we are handling.

This specific scenario's details is something that will be lost to time if we don't document it.

File: app/templates/flash-message-change-password-bug.html (hunk 1 above)
Comment (bartekn) [inline app/templates/flash-message-change-password-bug.html]: @litnerdy @bekkibolthouse do we have any FAQ entry about this? Could you post a final copy for this flash message here?

**Comment by litnerdy** [comment]
I think that copy looks fine for the flash message. Here's what's going to appear on the helpdesk:

"We had an error in our code (now resolved) that resulted in some users having broken recovery codes. If you see the message “Your recovery code has an error” in the client, please reset your code immediately."

**Comment by litnerdy** [comment]
Joyce is requesting an edit to the copy to "our recovery code must be reset"

**Comment by bartekn** [comment]
Please confirm it's OK now:

**Comment by bekkibolthouse** [comment]
Should read "Your" not "Our" at head of sentence.

**Comment by bartekn** [comment]
@litnerdy:
> "our recovery code must be reset"
Changing.

**Comment by bekkibolthouse** [comment]
Copy looks good, +1 on content.

**Pull request merged**